packages/itwinui-css/src/bridge.scss

```
@@ -96,6 +122,23 @@
     --iui-color-negative-hsl: 347.86deg 75.06% 76.69%; // --stratakit-color-icon-critical-faded
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Field */
+  .iui-field {
+    --_iui-field-color-background: var(--stratakit-color-bg-neutral-base);
```

Every set of overrides should have a meaningful comment, otherwise this file will become impossible to maintain.

```suggestion
    // StrataKit uses neutral colors for field, to differentiate from page.
    --_iui-field-color-background: var(--stratakit-color-bg-neutral-base);
```